packages/cli/yaml/validator/src/rules/valid-example-type/__test__/valid-example-type.test.ts

```diff
@@ -52,6 +52,24 @@ describe("valid-example-type", () => {
                 message: "Expected example to be a boolean. Example is: 100",
                 nodePath: ["types", "BooleanAlias", { key: "examples", arrayIndex: 1 }]
             },
+            {
+                severity: "error",
+                relativeFilepath: RelativeFilePath.of("alias.yml"),
+                message: 'Expected example to be a date. Example is: "2024-01-35"',
```

Comment:
Given that `2024-01-35` isn't a real date, want to swap to `2024-01-25`?

Reply:
that's part of the validation! we are erroring on invalid dates. a real date is 2024-01-01 (see in alias.yml L37) that notably does not have an expected error

Reply:
An _effective_ test case